.github/scripts/test_catch_up.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -eu
+
+pushd local-tests/
+
+if [ ! -f "$BINARY" ]; then
+  echo "Binary $BINARY does not exist."
+  exit 1
+fi
+
+echo 'Preparing environment'
+chmod +x $BINARY
+
+pip install -r requirements.txt
+
+echo 'Running test'
+./test_catch_up.py
```

Comment:
Minor: we can use `exec ./test_catch_up.py` to decrease proces child-parent stack (really for debugging purposes in case of some dangling test process etc)